src/include/utils/rangerrest.h

```diff
@@ -80,6 +80,13 @@ typedef struct RangerPrivilegeResults
 {
   RangerACLResult result;
   Oid relOid;
+
+  /* 
```

Comment:
On a *batch* ranger acl check request/response:
The acl check result order isn't the same as request.
When reponse's resource&privilege field is same as request, then we consider them is a pair.
 
So we need storage resource&privilege sign in here.